source/fundamentals/collations.txt

```diff
@@ -0,0 +1,280 @@
+==========
+Collations
+==========
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+This guide shows you how to use **collations**, a set of sorting rules, to
+run operations using string ordering for specific languages and locales (a
+community or region that shares common language idioms).
+
+MongoDB sorts strings using *binary collation* by default. This collation
+method uses the `ASCII standard <https://en.wikipedia.org/wiki/ASCII>`_
+character values to compare and order strings. Languages and locales
+have specific character ordering conventions that differ from the ASCII
+standard.
+
+For example, in Canadian French, the right-most accented character determines
+the ordering for strings when the other characters are the same. Consider the
+following French words: **cote**, **coté**, **côte**, and **côté**.
+
+The order in which they would appear using the default *binary
+collation* is:
+
+::
+
+   cote
+   coté
+   côte
+   côté
+
+If we apply the Canadian French collation, the order is:
+
+::
+
+   cote
+   côte
+   coté
+   côté
+
+.. note::
+
+   Collations are available in MongoDB 3.4 and later.
+
+-----
+Usage
+-----
+
+You can specify a collation when you create a **new collection** or
+**new index**. You can also specify a collation for :doc:`CRUD operations
+</fundamentals/crud>` and aggregations.
+
+When you create a **new collection** with a collation, you define the default
+collation for any of the :manual:`operations that support collation
+</reference/collation/#operations-that-support-collation>` called on that
+collection. You can override the collation for an operation by specifying a
+different one.
+
+When you create a **new index** with a collation, you specify the sort order
+for operations that use that index. To use the collation in the index, you
+must provide a matching collation in the operation, and the operation must
+use the index. While most index types support collation, the following
+types support only binary comparison:
+- :manual:`text </reference/method/db.collection.createIndex/#options-for-text-indexes>`
+- :manual:`2d </reference/method/db.collection.createIndex/#options-for-2dsphere-indexes>`
+- :manual:`geoHaystack </reference/method/db.collection.createIndex/#options-for-geohaystack-indexes>`
```

Comment:
It looks like your bulleted list here isn't rendering correctly.

Reply:
Weird, I had put a linebreak between 68-69 (the reason why it doesn't render correctly) but it's not there anymore. Thanks for catching this!